packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -665,3 +722,23 @@ module.exports = async (callback: (error?: any) => number) => {
     callback(error)
   }
 }
+
+function checkRGImplementationAbi(Contract: any, name: string) {
```

Comment:
I'm by no means a naming expert, but I would probably have the name indicate that you are actually returning the ABIs

Reply:
Good idea. I'll name this "checkAndReturnInitializationAbi"